### PR TITLE
Added image of 2048 to README.md

### PR DESCRIPTION
**Pull request by Juliush0604**
Google Code-in Task

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 This is a 2048 clone for [Haiku OS](https://haiku-os.org/).
```
Comment (humdingerb) [inline README.md]: It's just "Haiku", not "Haiku OS".

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
+![2048 screenshot](2048.png "2048")
+
 # Build
```
Comment (humdingerb) [inline README.md]: Remove the whole "Build" section.

**Comment by scottmc** [comment]
Please update or close this pull request.

**Comment by scottmc** [comment]
There's been a few changes to the progrm since the screenshot here was taken.  Also the pull request wasn't updated after changes were requested.  Closing since it will need to be redone anyways now.

**Pull request closed without merging**